Presentation.Web/Models/External/V2/Request/RightsHolderCreateItInterfaceRequestDTO.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Web;
+
+namespace Presentation.Web.Models.External.V2.Request
+{
+    public class RightsHolderCreateItInterfaceRequestDTO : RightsHolderWritableItInterfacePropertiesDTO
+    {
+        /// <summary>
+        /// UUID for owning organization
+        /// </summary>
+        /// <remarks>Use api/v2/rightsholder/organizations API for getting a list of possible organizations related to the logged in user</remarks>
+        [Required]
+        public Guid RightsHolderUuid { get; set; }
```

Comment:
Jeg er usikker på om modelstatevalidation fanger den her når den ikke er defineret for default(Guid) er Guid.Empty (Guid::=struct)

Reply:
Man kunne lave en ny attribute. DisallowNonEmptyGuid

Reply:
Eller et tjek i controlleren

Reply:
Løst ved tjek i controlleren